src/main/resources/templates/selection.html

```diff
@@ -37,7 +37,7 @@
                     <p>{#if it.show == it.localizedTitle}<b>{it.show}</b>{#else}<b>{it.localizedTitle}</b> ({it.show}){/if}</p>
                     <p>Staffel {it.number}<!-- von {it.totalSeasons}-->, {it.numberOfEpisodes} Episoden, {it.yearOfFirstAiring}</p>
                     <p><!-- letzte Episode gesehen am: {it.lastSeen}--></p>
-                    <p id="summary-{it.id}" class="summary">{it.summary}</p>
+                    <p id="summary-{it.id}" class="summary">{it.summary}<br><br><i>Nominiert von {it.nominatingUser}.</i></p>
```

Comment:
Consider not depending on `<br />` tags und using css margin or padding or flexbox and gap instaed. But if necessary I'd prefer to show that they are self closing by using `<br />` instead of `<br>`. I'd also prefer this to be split into multiple lines like:
```html
<p id="summary-{it.id}" class="summary">
    {it.summary}
    <br />
    <br />
    <i>Nominiert von {it.nominatingUser}.</i>
</p>
```